core/src/main/scala/org/apache/spark/SSLOptions.scala

```diff
@@ -149,12 +157,20 @@ private[spark] object SSLOptions extends Logging {
     val keyPassword = conf.getOption(s"$ns.keyPassword")
         .orElse(defaults.flatMap(_.keyPassword))
 
+    val keyStoreType = conf.getOption(s"$ns.keyStoreType")
```

Comment:
I see you adding these new properties but I don't see them used? Shouldn't `createJettySslContextFactory` and potentially `createAkkaConfig` be modified?